code/obj/critter/misc.dm

```diff
@@ -680,8 +680,11 @@
 		..()
 
 		src.visible_message("<span class='combat'><b>[src]</b> bursts into a puff of smoke!</span>")
-		var/datum/chemical_reaction/smoke/thesmoke = new
-		thesmoke.on_reaction(src.reagents, 12)
+		var/datum/reagents/temporary_holder = new/datum/reagents(1000)
+		src.reagents.copy_to(temporary_holder)
+		var/turf/T = get_turf(src)
+		smoke_reaction(src.reagents, 12, T)
+		classic_smoke_reaction(temporary_holder, 10, T)
```

Comment:
```suggestion
		classic_smoke_reaction(temporary_holder, 1, T)
```
I would not call a 10 tile radius a ***small*** aerosol.

Reply:
I'm not positive, but I think golems already bypass smoke blacklists as they are now. I'd rather use a direct call here because it allows the exact radius to be input (which is great for this, kind of happy that's a thing) if the end result is the same.

Reply:
Actually, 3 is better for this purpose than the 5 in the change I committed yesterday. The idea is to discourage kiting them like monkeys, 5 is more than needed. Not sure how to/if I can edit that in myself though